HISTORY.rst

```diff
@@ -1,7 +1,7 @@
 Release History
 ===============
 
-Unreleased Changes
+2.2.0 (2020-05-04)
```

Comment:
Hey it's the 14th, my friend. But may the fourth be with you. :)

Reply:
oops! fixed in 8b101f0